Test/Integration/Product/ProductsIndexingTest.php

```diff
@@ -1,24 +1,50 @@
 <?php
 
-namespace Algolia\AlgoliaSearch\Test\Integration;
+namespace Algolia\AlgoliaSearch\Test\Integration\Product;
 
 use Algolia\AlgoliaSearch\Model\Indexer\Product;
+use Algolia\AlgoliaSearch\Test\Integration\IndexingTestCase;
 use Magento\CatalogInventory\Model\StockRegistry;
+use Magento\Framework\Indexer\IndexerRegistry;
 
+/**
+ * @magentoDbIsolation disabled
+ * @magentoAppIsolation enabled
+ */
 class ProductsIndexingTest extends IndexingTestCase
 {
+    /** @var Product */
+    protected $productsIndexer;
+
+    /** @var StockRegistry */
+    protected $stockRegistry;
+
+    /*** @var IndexerRegistry */
+    protected $indexerRegistry;
+
     protected $testProductId;
 
+    public function setUp():void
+    {
+        parent::setUp();
+
+        $this->productsIndexer = $this->objectManager->get(Product::class);
+        $this->stockRegistry = $this->objectManager->get(StockRegistry::class);
+        $this->indexerRegistry = $this->objectManager->get(IndexerRegistry::class);
+
+        $priceIndexer = $this->indexerRegistry->get('catalog_product_price');
+        $priceIndexer->reindexAll();
+    }
+
     public function testOnlyOnStockProducts()
     {
+        $this->assertEquals(true, true);
```

Comment:
Is this just debugging code?

Reply:
Ah yes, forgot to remove that indeed, I will do it in the next PR ;)